README.md

```diff
@@ -123,3 +123,12 @@ FullStory.event('Subscribed', {
 const startOfPlayback = FullStory.getCurrentSessionURL();
 const playbackAtThisMomentInTime = FullStory.getCurrentSessionURL(true);
 ```
+
+### Sending custom page data
```

Comment:
nit: I haven't tested or read the docs/code carefully enough, but does the call just get thrown away if anything other than 'page' is passed in? If so maybe worth documenting that?

Reply:
![image](https://user-images.githubusercontent.com/12995427/151436809-ff7df09c-8ffe-4722-a6ca-8f39d23b80e9.png)

Typescript will prevent users from passing in any value other than "page". The type hints show that only allow "page" as a valid parameter. In this case it's not possible to make a call without "page".

Reply:
Yeah that's fair, I was thinking that if someone uses js vs. ts they wouldn't get type checked, but I guess that's unlikely.
Also since `pageName` is reserved field it might be good to call it out, but they are all documented in our help page so  maybe no need to duplicate the information here since that'd duplicate the document maintenance burden.

Reply:
I think a good middle ground would be to add a link to the help page on this readme. I will add that before merging in this PR.

Reply:
It might be a good idea to doubly-check with the internal team on what the behavior is in JS when an arbitrary string is passed in. @GGonryun

Reply:
Will add a comment in the documentation that "arbitrary strings in that parameter location will result in an Error that will either be logged to the browser console, or swallowed, depending on debug status of the FS Browser SDK" as [per this message](https://fullstory.slack.com/archives/C2CK80L1M/p1643320958385300?thread_ts=1643140203.321900&cid=C2CK80L1M).